src/lib/drivers/device/Device.hpp

```diff
@@ -184,6 +202,23 @@ class __EXPORT Device
 		uint32_t devid;
 	};
 
+	/**
+	 * Print decoded device id string to a buffer.
+	 *
+	 * @param buffer                        buffer to write to
+	 * @param length                        buffer length
+	 * @param id	                        The device id.
+	 * @param return                        number of bytes written
```

Comment:
The string is not always null-terminated. This should be handled and/or remarked.

Reply:
Explicitly null terminated